feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/rrm_config.c

```diff
@@ -0,0 +1,135 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <time.h>
+
+#include <uci.h>
+#include <uci_blob.h>
+
+#include <target.h>
+
+#include "ovsdb.h"
+#include "ovsdb_update.h"
+#include "ovsdb_sync.h"
+#include "ovsdb_table.h"
+#include "ovsdb_cache.h"
+
+#include "nl80211.h"
+#include "radio.h"
+#include "vif.h"
+#include "phy.h"
+#include "log.h"
+#include "evsched.h"
+#include "uci.h"
+#include "utils.h"
+#include "captive.h"
+
+ovsdb_table_t table_Wifi_RRM_Config;
+extern ovsdb_table_t table_Wifi_Radio_Config;
+extern ovsdb_table_t table_Wifi_VIF_Config;
+
+void rrm_config_vif(struct blob_buf *b, struct blob_buf *del, const char * freq_band, const char * if_name)
+{
+	struct schema_Wifi_RRM_Config conf;
+
+	memset(&conf, 0, sizeof(conf));
+	if (false == ovsdb_table_select_one(&table_Wifi_RRM_Config,
+			SCHEMA_COLUMN(Wifi_RRM_Config, freq_band), freq_band, &conf))
+	{
+		LOG(INFO, "Wifi_VIF_Config: No RRM for band %s", freq_band );
+		blobmsg_add_u32(del, "rssi_ignore_probe_request", -90);
+		blobmsg_add_u32(del, "signal_connect", -90);
+		blobmsg_add_u32(del, "signal_stay", -90);
+		blobmsg_add_u32(del, "bcn_rate", 0);
+		blobmsg_add_u32(del, "mcast_rate", 0);
+	} else {
+		blobmsg_add_u32(b, "rssi_ignore_probe_request", conf.probe_resp_threshold);
+		blobmsg_add_u32(b, "signal_connect", conf.client_disconnect_threshold);
+		blobmsg_add_u32(b, "signal_stay", conf.client_disconnect_threshold);
+		blobmsg_add_u32(b, "bcn_rate", conf.beacon_rate);
+		blobmsg_add_u32(b, "mcast_rate", conf.mcast_rate);
+	}
+	return;
+}
+
+static bool rrm_config_update( struct schema_Wifi_RRM_Config *conf, bool addNotDelete)
```

Comment:
addNotDelete not used anywhere. Can we get rid of that?

Reply:
Yes we should.